BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/DataConverter_PUEO.py

```diff
@@ -12,7 +12,11 @@
 
 #This preps the data to become a pandas dataframe
 #dtypedict={'InnerRadius':np.float64, 'Length':np.float64,'OpeningAngle':np.float64,'Fitness':np.float64,'Generation':np.int64,'GenerationString':str}
-df0 = pd.read_csv(g.location+"/runData.csv",skiprows=1,names=["SideLength","Height","XInitial","YInital","ZFinal","YFinal","Beta","Fitness","Generation"],index_col=False)
+df0 = pd.read_csv(g.location+"/runData.csv",skiprows=1,
+                  names=["SideLength","Height","XInitial",
+                         "YInital","ZFinal","YFinal","Beta",
+                         "TLength", "THeight","Fitness","Generation"],
+                  index_col=False)
```

Comment:
This cleans up the whitespace a bit:
```suggestion
df0 = pd.read_csv(g.location+"/runData.csv", skiprows=1,
                  names=["SideLength", "Height", "XInitial",
                         "YInital", "ZFinal", "YFinal", "Beta",
                         "TLength", "THeight", "Fitness", "Generation"],
                  index_col=False)
```

If you set the type of the `location` argument to `pathlib.Path`, then here you could use `g.location / 'runData.csv'` instead of  `g.location+"/runData.csv"`.  I think other scripts are already following the same approach and using `pathlib`.

Reply:
Cool, I added both suggestions in the new commit!